GEMINI 2.3 fixes

nmtoken opened the pull request:
Proposed changes to GEMINI 2.3.n schematron

nmtoken committed:
Add files via upload

nmtoken committed:
Merge pull request #1 from nmtoken/nmtoken-corrections-and-enhancements

Added a new schematron file

archaeogeek commented:
Any chance of a diff between this and the previous version?

nmtoken committed:
Added patch file for difference between current and new schematron

nmtoken commented:
Added a diff between the files, I uploaded a new file rather than change the existing one for two reasons, first is that the original file has a version number attached, and second related to that is that we may want a candidate release for all new proposed changes.  These changes should be for consideration/testing at the moment, not latest GEMINI.

PeterParslow commented:
I challenge the test that triggers MI-18a (line 33 in the diff)

test="($hierarchyLevelCLValue = 'dataset' or $hierarchyLevelCLValue = 'series') and ($srES &gt; 0 and $srES &gt; 0)

surely one of the two $srES should be $srD?

And you would then want to fix the report that it gives (line 35 in the diff)

nmtoken committed:
Update GEMINI_2.3_Schematron_Schema.sch

fixed bug in Item 18 - Spatial Resolution, removed commented out code Item 25 - Limitations on Public Access, added comment to head

nmtoken commented:
This is now out of date.
